src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -275,21 +234,16 @@ public void giveUserQuizTakenPoints_notScoreProperty() {
         user.setProperty("quiz_timestamp", 159430944365L);
         datastore.put(user);
 
-        boolean actual = false;
-        try{
-            actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
-        } catch(NullPointerException e) {
-            user.setProperty("score", 20);
-        }
-        Assert.assertEquals(user.getProperty("score"), 20);
+        boolean actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
+        user.setProperty("score", 20);
```

Comment:
This line should be removed, I think

Reply:
Like 238 is there to add the score entity because it was not previously there

Reply:
I will get rid of this line